src/main/resources/application.yaml

```diff
@@ -105,7 +105,10 @@ kodeverk:
   cache:
     gyldigTidMinutter: 60
 
-SAF_APP_CLIENT_ID: dev-fss.teamdokumenthandtering.saf
```

Comment:
Har denne forsvunnet?

Reply:
Nei, den er bare flyttet noen linjer ned for å gruppere client ids.

Reply:
Jeg endret DOKARKIV_CLIENT_ID til api.client.dokarkiv.clientId slik at den ligger i samme struktur som url'en

Reply:
Flott!